pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -117,6 +116,17 @@ func splitHostIntPort(s string) (string, int, error) {
 	return host, portInt, err
 }
 
+func validatePercentageOfNodesToScore(percentageOfNodesToScore *int32) []error {
```

Comment:
Re https://github.com/kubernetes/kubernetes/pull/112521#discussion_r977201934, you should have `path` as its first parameter, otherwise for the profile-level validation, the error message on path is incorrect

Reply:
Made the change.